core/bentley/src/MobileUtils.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/** @internal */
+export class MobileUtils {
+  public static get isITwinFrontend() {
+    return (typeof window === "object" && typeof window.navigator === "object") && undefined !== (window as any).itwin;

```

Comment:
What is `isITwinFrontend` used for? A while ago @calebmshafer made it possible to use imodeljs-frontend outside of a browser context for some user. In that case, `window` will be undefined so `isITwinFrontend` will be false.

Reply:
This is so we can tell the difference between our frontend, started by our app, and a mobile browser outside of our app. If window isn't defined, MobileUtils.isItwinFrontend will be false, as it should be.

@swbsi  will have to make sure we set it when we start our frontend process from mobile apps.

Reply:
see if you like the current approach better.